extensions/wikia/ArticleVideo/bower_components/html5-skin/tests/controller-test.js

```diff
@@ -124,17 +130,16 @@ OO = {
           removeClass: function(a) {}
         },
         mainVideoElement: {
-          addClass: function(a) {},
-          removeClass: function(a) {},
-          get: function(a) {
-            return {
-              webkitSupportsFullscreen: true,
-              webkitEnterFullscreen: function() {},
-              webkitExitFullscreen: function() {},
-              addEventListener: function(a,b) {}
-            }
-          }
-        }
+          classList: {
+            add: function(a) {},
+            remove: function(a) {}
+          },
+          webkitSupportsFullscreen: true,
+          webkitEnterFullscreen: function() {},
+          webkitExitFullscreen: function() {},
+          addEventListener: function(a,b) {}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "a". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "b". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)